### DISCUSSION TITLE
ReferenceError: process is not defined

### DISCUSSION THREAD

**Discussion by roylans** [quasarframework/quasar#12171] [General - Components / Directives / etc] [answered]
Hi friends

I have created a new project but when I try to use 

const api = axios.create({ baseURL: process.env.API })

in boot/axios.ts

I have this issue: ReferenceError: process is not defined

any ideas?

Thanks in advance

**Comment by yusufkandemir** [comment, marked as answer]
It is happening because the `API` env variable is not set. Make sure you are providing it in `quasar.conf.js > build > env` as shown in https://quasar.dev/quasar-cli/handling-process-env#adding-to-process-env. Make sure the value you are passing is actually set, either in the .env file if you are using dotenv, in your system's environment variable list(_see `printenv` for unix-like systems, `SET` for Windows_), or provided directly per command(_e.g. `API=http://my-api.localhost quasar dev`_).

Also see:
#12061
#10088

**Reply by roylans** [reply]
Thanks solved
